server/echo_handler.cc

```diff
@@ -0,0 +1,19 @@
+#include "echo_handler.h"
+#include "httpRequest.h"
+#include "httpResponse.h"
+#include "http.h"
+#include <string>
+
+EchoHandler::Status EchoHandler::Init(const std::string& uri_prefix,
+				      const NginxConfig& config) {
+  return OK;
+}
```

Comment:
I can't think of anything to init. I believe we said that the server already validates the uri_prefix, and I suppose I could add more checking the number of tokens in the config